packages/scaffold-config/test/unit/detect.spec.ts

```diff
@@ -144,22 +144,8 @@ describe('detectFramework', () => {
     const projectPath = await scaffoldMigrationProject('react-vite-ts-unconfigured')
 
     fakeDepsInNodeModules(projectPath, [
-      { devDependency: 'vite', version: '2.0.0' },
-      { dependency: 'react', version: '17.0.0' },
-    ])
-
-    const actual = await detectFramework(projectPath, resolvedCtFrameworks)
-
-    expect(actual.framework?.type).to.eq('react')
-    expect(actual.bundler).to.eq('vite')
-  })
-
-  it(`React with Vite using pre-release version`, async () => {
```

Comment:
we support installing a pre release if we support that major version, which is kind of weird because we don't know if we will be compatible with the next major version, so we shouldn't actually support this OOTB and instead let users skip the install step to try the test harness